tasks/deploy-flatlaunchpeg.ts

```diff
@@ -16,6 +16,14 @@ task('deploy-flatlaunchpeg', 'Deploy FlatLaunchpeg contract')
 
     const launchConfig = loadLaunchConfig(configFilename)
 
+    // This is used for testing purposes
+    if (launchConfig.allowlistStartTime === 'Soon') {
+      launchConfig.allowlistStartTime = Math.floor(Date.now() / 1000) + 120
+    }
+    if (launchConfig.publicSaleStartTime === 'Soon') {
+      launchConfig.publicSaleStartTime = launchConfig.allowlistStartTime + 120
+    }
```

Comment:
Can we move this logic in `utils.ts` ? I think it will improve the readability of the deploy task. It could be a function `convertTimestampIfNeeded` that we call in `loadLaunchConfig`.

Since `allowlistStartTime` depends on whether it's a launchpeg or flat launchpeg, we can check if launchConfig.auctionSaleStartTime is defined and set `allowlistStartTime` accordingly